frontend/src/index.pug

```diff
@@ -131,7 +131,10 @@ html
           .summary-charts(v-for="repo of filteredRepos")
             .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'") {{ repo[0].repoName }}&nbsp;&nbsp;
               .summary-charts__title--contribution(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
-            .summary-charts__title(v-if="filterGroupSelection === 'groupByAuthors'") {{ repo[0].displayName }} ({{ repo[0].name }})&nbsp;&nbsp;
+            .summary-charts__title(
+                v-if="filterGroupSelection === 'groupByAuthors'",
+                v-bind:class="{ red: repo[0].name === '-' }"
```

Comment:
The class name `red` is a bit weird. But I cannot find a better one myself.

Reply:
how about `warning`?

Reply:
Can you check on the `.warn` class of CSS? `.warning` is also better than `red`.